### PR TITLE
Add quickstarts reference

### PR DESCRIPTION
**Pull request by rezabekf**
*Issue #, if available:*
N/A
*Description of changes:*
This PR adds QuickStarts section and reference links to their content.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by tbulding — DISMISSED** [review]
I would suggest replacing "further" in the Next Steps section with "additional" or "more". This is only a suggestion

File: workshop/content/next-steps/_index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Next Steps"
+date: 2022-02-28T09:42:46Z
+weight: 60
+
+---
+
+In this section, you can find further content related to CloudFormation, such as template examples, solutions to common
```
Comment (rezabekf) [inline workshop/content/next-steps/_index.md:8]:
```suggestion
In this section, you can find additional content related to CloudFormation, such as template examples, solutions to common
```

Comment (rezabekf) [reply workshop/content/next-steps/_index.md:8]: Done. Thank you for the feedback!

File: workshop/content/next-steps/_index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Next Steps"
+date: 2022-02-28T09:42:46Z
+weight: 60
+
+---
+
+In this section, you can find additional content related to CloudFormation, such as template examples, solutions to common
+architecture designs, tools and more...
```
Comment (rezabekf) [inline workshop/content/next-steps/_index.md:9]:
```suggestion
In this section, you can find additional content related to AWS CloudFormation, such as template examples, solutions to common  architecture designs, tools, and more...
```

File: workshop/content/next-steps/aws-quick-starts/_index.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Aws Quick Starts"
```
Comment (rezabekf) [inline workshop/content/next-steps/aws-quick-starts/_index.md:2]:
```suggestion
title: "AWS Quick Starts"
```

**Review by mrinaudo-aws — APPROVED** [review]
lgtm

**Review by troy-ameigh — APPROVED** [review]
Thank you, for this.

**Pull request merged**